pkg/kubelet/metrics/collectors/resource_metrics_test.go

```diff
@@ -44,13 +44,16 @@ func (m *mockSummaryProvider) GetCPUAndMemoryStats() (*statsapi.Summary, error)
 }
 
 func TestCollectResourceMetrics(t *testing.T) {
-	testTime := metav1.NewTime(time.Unix(2, 0)) // a static timestamp: 2000
+	// a static timestamp: 2021-06-23 05:11:18.302091597 +0800
+	staticNonaTimestamp := time.Unix(0, 1624396278302091597)
```

Comment:
What is Nona?

Reply:
Sorry, it is a misstake, I mean to use nanoseconds, updated